### PR TITLE
Carousel component

### PR DESCRIPTION
**Pull request by siddhi1201**

### CODE REVIEW

File: components/slideshow.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import Image from 'next/image';
+import { useState } from 'react';
+import styles from './Slideshow.module.css';
+
+const Slideshow = (): JSX.Element => {
```
Comment (GetPsyched) [inline components/slideshow.tsx:5]:
```suggestion
export default function Slideshow() {
```

File: components/slideshow.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          />
+        </div>
+      ))}
+      <a className={styles.prev} onClick={prevSlide}>&#10094;</a>
+      <a className={styles.next} onClick={nextSlide}>&#10095;</a>
+    </div>
+  );
+};
+
+export default Slideshow;
```
Comment (GetPsyched) [inline components/slideshow.tsx:40]:
```suggestion

```

**Review by GetPsyched — CHANGES_REQUESTED** [review]
Great stuff for learning NextJS for the first time. Hope the use of LLMs is minimal to none :P

Resolve the changes mentioned and use TailwindCSS instead of CSS modules.

And it looks like you have tested the changes outside of the project, if that's not the case, great, else test it by running `npm run dev`

**Comment by siddhi1201** [comment]
Thank you so much :)
will make the changes accordingly.

File: components/slideshow.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    </div>
+  );
+};
+
+
+
+
+
+
+
```
Comment (GetPsyched) [inline components/slideshow.tsx:45]:
```suggestion

```

File: components/slideshow.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+          <Image
+            src={slide}
+            alt={"slides ${index + 1}"}
+            width={1920}
+            height={1080}
+            loading={index === 0 ? 'eager' : 'lazy'} 
+          />
+        </div>
+      ))}
+      <button className={styles.prev} onClick={prevSlide}>&#10094;</button>
```
Comment (GetPsyched) [inline components/slideshow.tsx:34]: The button for the previous image should come before the slides mapping. Would need to adjust the styling accordingly.

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: components/slideshow.tsx (hunk 3 above)
Comment (GetPsyched) [reply components/slideshow.tsx:45]: Please run `npm run format` and add the changes before committing.

Comment (siddhi1201) [reply components/slideshow.tsx:45]: okay will take care

File: components/slideshow.tsx (hunk 4 above)
Comment (siddhi1201) [reply components/slideshow.tsx:34]: Okay. Will make the changes accordingly.

**Comment by geekysilento** [comment]
@GetPsyched ShadCN already has a carousel component, shouldn't we just use that for consistency plus its pre-styled? 
https://ui.shadcn.com/docs/components/carousel

**Comment by GetPsyched** [comment]
Hmm, that's true. Let's do that as a refactor later instead.

File: components/slideshow.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import Image from 'next/image';
+import { useState } from 'react';
+export default function Slideshow (){
```
Comment (GetPsyched) [inline components/slideshow.tsx:3]: Are you running `npm run format`? Wonder how are changes being committed without formatting

File: components/slideshow.tsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  return (
+    <div className="relative">
+      <button className="prev absolute top-1/2 transform -translate-y-1/2 opacity-0 transition-opacity duration-300 hover:opacity-100" onClick={prevSlide}>
+        &#10094;
+      </button>
+      <button className="next absolute top-1/2 transform -translate-y-1/2 opacity-0 transition-opacity duration-300 right-0 hover:opacity-100" onClick={nextSlide}>
+        &#10095;
+      </button>
+      {slides.map((slide: string, index: number) => (
+        <div className={index === currentSlide ? 'mySlides' :'mySlides hidden' } key={index}>
```
Comment (GetPsyched) [inline components/slideshow.tsx:28]: What's `mySlides` classname?

**Review by GetPsyched — CHANGES_REQUESTED** [review]

**Review by GetPsyched — COMMENTED** [review]
You still haven't tested this alongside our code. It doesn't even run.

Anyways, I'm hijacking this PR now.

**Review by GetPsyched — CHANGES_REQUESTED** [review]
Please run `npm i` and `npm run dev` to see the website and test changes. Also, use react-icons for the arrows, not unicode.

**Review by GetPsyched — CHANGES_REQUESTED** [review]
Why are you running `npx create-next-app` inside `slideshow/`? I'm so confused. All you need to do it run `npm run dev`

**Comment by siddhi1201** [comment]
Some commands weren't running, so I thought this might help. But later, I only used npm run dev.

**Comment by GetPsyched** [comment]
If any command isn't running, reach out to us and we'll help you out.

**Comment by GetPsyched** [comment]
Re-assigning this task to @som-04

**Pull request closed without merging**